client/src/actions/bookActions.js

```diff
@@ -7,7 +7,8 @@ export const getBooks = () => dispatch => {
   // };
   dispatch(setBooksLoading());
   axios
-    .get('/api/books')
+    // .get('/api/books')
+    .get('/books')
```

Comment:
I'm not an expert with `axios` but I think you are using it wrong here. I take a look at the documentation and you don't need to dispatch actions here. If you pass the correct parameters in the action, `axios` will do all the requests and dispatch all the actions that you need. Maybe you have to take a look and refactor this actions because it might be improved.